sql/promscale--0.0.0.sql

```diff
@@ -372,6 +374,27 @@ END;
 $block$
 ;
 
+DO $block$
+DECLARE
+    _rec record;
+BEGIN
+    FOR _rec IN
+    (
+        SELECT m.*
+        FROM _prom_catalog.metric m
+        order by m.table_schema, m.table_name
+    )
+    LOOP
+        EXECUTE format($sql$ALTER TABLE %I.%I OWNER TO %I$sql$, _rec.table_schema, _rec.table_name, current_user);
```

Comment:
This should probably only happen iff table_schema = 'prom_data'. Other metrics are continuous aggregates created and controlled by the user